app/components/Form/ProjectEmissionIntensityReportForm.tsx

```diff
@@ -24,6 +24,46 @@ interface Props {
   onSubmit: () => void;
 }
 
+// passing content suffix to fields that need nominator and denominator (example: tCO2e/GJ)
+export const createEmissionIntensityReportUiSchema = ({
+  emissionFunctionalUnit,
+  productionFunctionalUnit,
+}) => {
+  // Example: tCO2e/GJ if we have emissionFunctionalUnit
+  const contentSuffix: JSX.Element = emissionFunctionalUnit && (
+    <b>{`${emissionFunctionalUnit}${
+      productionFunctionalUnit ? `/${productionFunctionalUnit}` : ""
+    }`}</b>
+  );
+  return {
+    ...emissionIntensityReportUiSchema,
+    baselineEmissionIntensity: {
+      ...emissionIntensityReportUiSchema.baselineEmissionIntensity,
+      "ui:options": {
+        contentSuffix,
+      },
```

Comment:
Alternatively, for something more concise, you could do a deep copy of the schema and set the properties where needed (I'll leave the implementation up to you, whatever you feel is more readable/testable)

```
const uiSchemaCopy = JSON.parse(JSON.stringify(emissionIntensityReportUiSchema));

uiSchemaCopy.baselineEmissionIntensity["ui:options"].contentSuffix = contentSuffix;
uiSchemaCopy.targetEmissionIntensity["ui:options"].contentSuffix = contentSuffix;
//...etc...

return uiSchemaCopy;

Reply:
@pbastia This part didn't work because we don't have `["ui:options"]` in the UISchema so we can't add `contentSuffix` to that:
```
uiSchemaCopy.baselineEmissionIntensity["ui:options"].contentSuffix = contentSuffix;
uiSchemaCopy.targetEmissionIntensity["ui:options"].contentSuffix = contentSuffix;
//...etc...
```

this is the alternative:
```
uiSchemaCopy.baselineEmissionIntensity["ui:options"] = {
    ...uiSchemaCopy.baselineEmissionIntensity["ui:options"],
    contentSuffix,
  };
  uiSchemaCopy.targetEmissionIntensity["ui:options"] = {
    ...uiSchemaCopy.targetEmissionIntensity["ui:options"],
    contentSuffix,
  };
...
```

Reply:
👌